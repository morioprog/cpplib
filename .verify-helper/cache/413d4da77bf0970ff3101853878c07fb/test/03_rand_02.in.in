20 42
4 19
4 10
4 7
6 2
6 8
6 14
11 12
11 9
11 0
11 3
2 9
2 18
12 8
8 19
8 18
9 19
9 16
9 5
9 1
19 0
19 1
19 18
0 10
0 15
0 1
16 5
16 3
16 15
16 7
10 3
10 15
10 1
10 7
5 3
5 15
5 1
17 15
15 1
15 7
1 18
1 7
18 7
