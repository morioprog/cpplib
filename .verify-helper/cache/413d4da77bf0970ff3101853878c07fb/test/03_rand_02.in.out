17
13
11
12
6
14
8
4
2
9
16
5
19
0
10
15
1
18
7
3
